gitea/release.go

```diff
@@ -13,20 +13,21 @@ import (
 
 // Release represents a repository release
 type Release struct {
-	ID           int64         `json:"id"`
-	TagName      string        `json:"tag_name"`
-	Target       string        `json:"target_commitish"`
-	Title        string        `json:"name"`
-	Note         string        `json:"body"`
-	URL          string        `json:"url"`
-	TarURL       string        `json:"tarball_url"`
-	ZipURL       string        `json:"zipball_url"`
-	IsDraft      bool          `json:"draft"`
-	IsPrerelease bool          `json:"prerelease"`
-	CreatedAt    time.Time     `json:"created_at"`
-	PublishedAt  time.Time     `json:"published_at"`
-	Publisher    *User         `json:"author"`
-	Attachments  []*Attachment `json:"assets"`
```

Comment:
Yes, `attachements` should not be removed here.

Reply:
See discussion in #83

Reply:
I probably should have commented in this PR about why I removed it, so apologies for miscommunication on my part.